configs/Wisun_Stm_s2lp_RF.json

```diff
@@ -0,0 +1,67 @@
+{
+    "config": {
+        "heap-size": {
+             "help": "The amount of static RAM to reserve for nsdynmemlib heap",
+             "value": 100000
+        },
+        "radio-type":{
+            "help": "options are ATMEL, MCR20, SPIRIT1, S2LP",
+            "value": "S2LP"
+        },
+        "backhaul-driver": {
+            "help": "options are ETH, SLIP, EMAC",
+            "value": "EMAC"
+        },
+        "mesh-mode": {
+            "help": "Mesh networking mode. Options are LOWPAN_ND, LOWPAN_WS and THREAD",
+            "value": "LOWPAN_WS"
+        },
+        "backhaul-mac-src": {
+            "help": "Where to get EUI48 address. Options are BOARD, CONFIG",
+            "value": "BOARD"
+        },
+        "backhaul-mld": {
+            "help": "Enable proxying Multicast Listener Discovery messages to backhaul network",
+            "value": "false"
+        },
+        "backhaul-mac": "{0x02, 0x00, 0x00, 0x00, 0x00, 0x01}",
+        "debug-trace": "false",
+        "backhaul-dynamic-bootstrap": true,
+        "backhaul-prefix": "fd00:db8:ff1::",
+        "backhaul-default-route": "::/0",
+        "backhaul-next-hop": "fe80::1",
+        "multicast-addr": "ff05::7",
+        "LED": "NC",
+        "SERIAL_TX": "NC",
+        "SERIAL_RX": "NC",
+        "SERIAL_CTS": "NC",
+        "SERIAL_RTS": "NC",
+        "uc-channel-function": 255,
+        "bc-channel-function": 255,
+        "regulatory-domain": 3,
+        "operating-class": 255,
+        "operating-mode": 255,
+        "uc-fixed-channel": "0xffff",
+        "bc-fixed-channel": "0xffff",
+        "network-name": "\"ARM-WS-TESTING\""
+    },
+    "target_overrides": {
+        "*": {
+            "target.network-default-interface-type": "ETHERNET",
+            "nsapi.default-stack": "NANOSTACK",
+            "mbed-trace.enable": 1,
+            "nanostack.configuration": "ws_border_router",
+            "platform.stdio-convert-newlines": true,
+            "platform.stdio-baud-rate": 115200,
```

Comment:
Can we use faster baudarate. This slow ass speed

Reply:
The same stdio-baud-rate is used pretty much in all our applications? Is there some need for higher baud-rate?

Reply:
we typically use 460800 it will not buffer so much in large printing storm.

Reply:
Ok, all the other configs in nanostack-border-router are also using 115200 so for now I would like to stick with this and possibly change all of the configs in a separate PR.

Reply:
we have use 1MB spped even small atmel avr so why ARM not like to use same.

Reply:
If you can point me to the table that show what baud rates all 100 Mbed boards support, then OK to change. Otherwise do not touch.

(ps. such a table does not exist. 115200 is pretty much standard and Greentea uses 9600)

Reply:
Maybe driver team should validate UART driver which speed is supported. I think that 115200 is a bottle neck when we trace a lot.

Reply:
Don't trace a lot...
And there is no UART tests in Mbed OS. so there is nobody to verify the speeds that board supports. Sorry.

Reply:
Btw.. You should turn on buffered stdio.
It helps the speed, as it handles buffering in the UART interrupt, so tracing does not block, unless out of buffer.

Reply:
Good catch, that is missing also from the other configs for nanostack-border-router as well.

Reply:
Actually it was there already for wi-sun configuration, it is missing from the other configurations.